scripts/git_page/md_post_process.py

```diff
@@ -117,24 +117,25 @@ def remove_subheaders_after_first(md_path: Union[str, Path]) -> None:
     """
     try:
         with Path(md_path).open('r+') as md_file:
-            lines = [s.strip() for s in md_file.readlines()]
```

Comment:
the contents of `lines` will be used to rewrite the file below so the `\n` should be stripped away